pkg/apihelpers/apihelpers.go

```diff
@@ -71,6 +71,17 @@ var (
 					},
 				},
 			},
+			{
+				Path: constants.CrioPoliciesDir,
```

Comment:
Let's add this to the MCO docs https://github.com/openshift/machine-config-operator/blob/b6d267419cac5c8a580836d7d3555b8006faeb40/docs/MachineConfigDaemon.md#rebootless-updates Can be in a follow up PR.